components/infra-proxy-service/server/proxy.go

```diff
@@ -141,3 +144,14 @@ func DefaultIfEmpty(value string) string {
 
 	return value
 }
+
+// StructToJSON convert the structpb to JSON interface object.
+func StructToJSON(data *structpb.Struct) (interface{}, error) {
```

Comment:
Dup of https://github.com/chef/automate/pull/3871/files#diff-b64e5871e0080c88c1eacf4ac460a6c5R148-R157 will rebase once PR https://github.com/chef/automate/pull/3871 get merge.